packages/salesforcedx-vscode-soql/CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# 50.14.0 - January 21, 2021
+
+## Added
+
+- SOQL code-completion on WHERE clause expressions ([PR #2839](https://github.com/forcedotcom/salesforcedx-vscode/pull/2839))
+- Experimental option to perform remote query validation ([PR #2847](https://github.com/forcedotcom/salesforcedx-vscode/pull/2847))
+- Do not emit message from custom editor to webview if SOQL has not changed ([PR #2818](https://github.com/forcedotcom/salesforcedx-vscode/pull/2818))
+- Running query indicator ([PR #2822](https://github.com/forcedotcom/salesforcedx-vscode/pull/2822))
```

Comment:
Not sure what this one means/does

Reply:
This could be reworded as something like "Display progress notification when query is running"

Reply:
fixed